ecc/goldilocks/constants.go

```diff
@@ -1,6 +1,19 @@
 package goldilocks
 
-import fp "github.com/cloudflare/circl/math/fp448"
+import (
+	"errors"
+
+	fp "github.com/cloudflare/circl/math/fp448"
+)
+
+const (
+	// ScalarSize is the size (in bytes) of scalars.
+	ScalarSize = 56
+	// CurveEncodingSize is the size (in bytes) of an encoded point on the Goldilocks curve.
+	CurveEncodingSize = fp.Size + 1
+	// DecafEncodingSize is the size (in bytes) of an encoded Decaf element.
+	DecafEncodingSize = fp.Size
+)
 
 var (
```

Comment:
Nit: a link to a place in the original documents where these values actually come from